docs/tutorials/Aragon_Core_User_Guide.md

```diff
@@ -14,21 +14,21 @@ In addition to a working laptop or desktop computer with a decent internet conne
 
 #### Step 1. Navigate to [app.aragon.one](http://app.aragon.one) in your web browser and click “Create a new organization”
```

Comment:
We should use http**s**://app.aragon.one

Reply:
resolved